vtm/src/org/oscim/core/MapElement.java

```diff
@@ -31,6 +34,8 @@
  */
 public class MapElement extends GeometryBuffer {
 
+    public List<GeoPoint> geoPoints = new ArrayList<>();
```

Comment:
What exactly does it store?

Such mix doesn't seem nice or performant, i.e. `MapElement` to hold a `List` of...